examples/erc20-react-example/src/App.tsx

```diff
@@ -6,7 +6,7 @@ import {
   Sygma,
   EvmBridgeSetupList,
   BridgeEvents,
-} from "@buildwithsygma/sygma-sdk-core";
+} from "@buildwithsygma/sygma-sdk-core/deprecated";
```

Comment:
This means that the example is still going to use Sygma class? Is there an issue to update the examples?

Reply:
Readmes updated